src/main/java/com/googleinterns/zoomtube/servlets/DiscussionServlet.java

```diff
@@ -52,6 +52,15 @@
   @VisibleForTesting static final String PARAM_TIMESTAMP = "timestamp";
   @VisibleForTesting static final String PARAM_TYPE = "type";
 
+  @VisibleForTesting static final String ERROR_MISSING_LECTURE = "Missing lecture parameter.";
+  @VisibleForTesting
+  static final String ERROR_MISSING_COMMENT_TYPE = "Missing comment type parameter.";
```

Comment:
Did the linter cause this?

Reply:
Yeah must have... I'll change to `ERROR_MISSING_TYPE` so it fits on a line

Reply:
Actually nvm, that wouldn't cause it to fit